internal/record/recording_https_proxy.go

```diff
@@ -70,16 +70,22 @@ func (r *RecordingHTTPSProxy) handleRequest(w http.ResponseWriter, req *http.Req
 	}
 	fmt.Printf("Recording request: %s %s\n", req.Method, req.URL.String())
 
-	reqHash, err := r.recordRequest(req)
-	if err != nil {
-		fmt.Printf("Error recording request: %v\n", err)
-		http.Error(w, fmt.Sprintf("Error recording request: %v", err), http.StatusInternalServerError)
+	recReq, recErr := r.recordRequest(req)
```

Comment:
nit: calling the error variable `err` is pretty common

Reply:
Done, I didn't know we could re-assign new errors in the same code block